pkgs/development/haskell-modules/with-packages-wrapper.nix

```diff
@@ -1,7 +1,11 @@
-{ stdenv, ghc, packages, buildEnv, makeWrapper, ignoreCollisions ? false }:
+{ stdenv, ghc, llvmPackages, packages, buildEnv
+, makeWrapper
+, ignoreCollisions ? false, withLLVM ? false }:
+
+with stdenv.lib;
```

Comment:
IMHO, this change is unrelated and I'd rather not merge it (and the other changes that depend on it) as part of the LLVM updates. I realize it's a matter of taste, but I'd rather not have symbols in scope if I don't have to; I tend to avoid  using`with`.